### DISCUSSION TITLE
Marking same project complete for different roadmap

### DISCUSSION THREAD

**Discussion by amilliiee** [devchallenges-io/curriculum#11] [Report a bug] [answered]
Using the multi-step register form as an example: the project was completed for the responsive web design roadmap, and reappears for the JS roadmap, but doesn't show as complete in the JS roadmap.

**Comment by nghiemthu** [comment]
I am not sure I get your question, `multi-step register form` is in JS roadmap. It's not in `responsive web design roadmap`. Do you want it to appear in `responsive web design roadmap`?

**Reply by amilliiee** [reply]
Ah, I see where I made a mistake. The multi-step register form challenge in the JS roadmap links to the contact page challenge that is completed within the responsive web design roadmap. I completely missed that the URL didn't match the name of the challenge.

![inc-challenge](https://github.com/user-attachments/assets/56defdf4-dfb3-4ecc-8e37-476f15e15bf1)

**Reply by nghiemthu** [reply, marked as answer]
Oh then that's my mistake, it should have the correct link now. Thank you for reporting the bug 🙏
